stablehlo/transforms/StablehloRefineShapes.cpp

```diff
@@ -55,6 +55,33 @@ namespace stablehlo {
 
 namespace {
 
+// DenseElementsAttr can be constructed from ArrayRef<APInt> but not from
+// ArrayRef<APSInt>. This helper bridges the gap.
+DenseIntElementsAttr getTensorAttr(ShapedType type, ArrayRef<APSInt> values) {
+  SmallVector<APInt> supportedValues;
```

Comment:
Does the vector constructor allow coercion?

```
SmallVector<APInt> supportedValues(values.begin(), values.end());
```

Reply:
Nice! It indeed does.